stetho/src/main/java/com/facebook/stetho/inspector/elements/android/window/WindowRootViewCompactV16Impl.java

```diff
@@ -0,0 +1,57 @@
+package com.facebook.stetho.inspector.elements.android.window;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.view.WindowManager;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class WindowRootViewCompactV16Impl extends WindowRootViewCompat {
+	private Context mContext;
+
+	WindowRootViewCompactV16Impl(Context context) {
+		this.mContext = context;
+	}
+
+	@NonNull
+	@Override
+	public List<View> getRootViews() {
+		WindowManager windowManager = (WindowManager) mContext.getSystemService(Context.WINDOW_SERVICE);
+		if (windowManager == null) {
+			throw new RuntimeException("unfortunately, you cannot view the view tree of the dialog etc. windowManager == null !");
```

Comment:
should I remove this if statement?

Reply:
Well, I can't really tell why you added it in the first place, but yes I generally would feel comfortable proceeding assuming that the windowManager is not null.  I can't fathom what it could possibly mean for that to be the case...

Reply:
Because  there is a warning in android studio   Argument 'windowManager' might be null